.github/workflows/test-opencast-build.yml

```diff
@@ -73,6 +73,17 @@ jobs:
             -Dmaven.wagon.http.pool=false \
             -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
 
+        #This has to happen in a separate step or else various service classes aren't visible
+      - name: build javadocs
+        run: |
+          mvn javadoc:javadoc -Pnone \
+            --batch-mode \
+            -Dsurefire.rerunFailingTestsCount=2 \
+            -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn \
+            -Dhttp.keepAlive=false \
+            -Dmaven.wagon.http.pool=false \
+            -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
```

Comment:
Do we need all those arguments for building the javadocs?

Reply:
Likely not, but they're also not going to hurt.  The only major change is allowing failing test counts > 0, to help with the admin ui test flakiness.  The `maven.wagon` and `http` stuff helps with random http issues experienced inside GHA's environments.